### PR TITLE
feat: add action.process_report

### PR DESCRIPTION
**Pull request by revolunet**
Hi,

Thanks for this great action;

I'd like to propose to add the ability to disable the issue creation. this may not always be needed

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by kingthorin** [comment]
Please update README.md with the new details, and add a note to the unreleased section of CHANGELOG.md.

File: index.js [hunk 1]
```diff
@@ -58,7 +59,12 @@ async function run() {
                 console.log('Scanning process completed, starting to analyze the results!')
             }
         }
-        await common.main.processReport(token, workspace, plugins, currentRunnerID, issueTitle, repoName);
+        if (String(processReport).toLowerCase() === 'true') {
+            await common.main.processReport(token, workspace, plugins, currentRunnerID, issueTitle, repoName);
+        } else {        
+            console.log('[WARNING]: \'process_report\' is set to \'false\', issues wont be created');
```
Comment (kingthorin) [inline index.js:65]:
```suggestion
            console.log('[WARNING]: \'process_report\' is set to \'false\' (or an invalid value), issues wont be created');
```

**Comment by revolunet** [comment]
fixed !

**Comment by psiinon** [comment]
I could be wrong but I thought you could disable issue creation by just not defining the token - doesnt that have the same effect? If so I dont think we should have a second way of doing this, just maybe improved docs ;)

**Comment by revolunet** [comment]
Ho maybe, if its the case then it should be enough indeed

**Comment by revolunet** [comment]
I tried by commenting the token in my action but for some reason, it still creates issues

**Comment by thc202** [comment]
Try pass an empty string instead, the default value is the GitHub token.

**Comment by revolunet** [comment]
i can confirm that setting explicitly `token: ""` disable issue creation. thanks @thc202

**Comment by revolunet** [comment]
While this disable issue création, this result in a "Bad credentials" error in the job

**Pull request closed without merging**